docraptor.gemspec

```diff
@@ -1,4 +1,26 @@
 # -*- encoding: utf-8 -*-
+#
+=begin
+#DocRaptor v1
+
+OpenAPI spec version: 1.1.0
+
+Generated by: https://github.com/swagger-api/swagger-codegen.git
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=end
+
```

Comment:
We already have [`post_generate_language`](https://github.com/DocRaptor/docraptor-ruby/blob/master/script/post_generate_language) in this repo, but we'd need something like [our time annotation remover in Java](https://github.com/DocRaptor/docraptor-java/blob/master/script/post_generate_language#L5-L7).